gp/hl_api.py

```diff
@@ -1,18 +1,21 @@
 import numpy as np
+from typing import Optional, Callable
+from .abstract_individual import AbstractIndividual
+from .abstract_population import AbstractPopulation
+from .ea import MuPlusLambda
 
 
 def evolve(
-    pop,
-    objective,
-    ea,
-    max_generations,
-    min_fitness,
-    print_progress=False,
-    *,
-    callback=None,
-    label=None,
-    n_processes=1,
-):
+    pop: AbstractPopulation,
+    objective: Callable[[AbstractIndividual], AbstractIndividual],
+    ea: MuPlusLambda,
```

Comment:
now this would be a reason to create an abstract ea class ;)

Reply:
Yes, that's what I've been saying in #33

Reply:
right :D ok, let's stick to our decision and keep it like it is for now